test/powershell/Modules/Microsoft.PowerShell.Utility/Export-FormatData.Tests.ps1

```diff
@@ -14,33 +21,70 @@ Describe "Export-FormatData DRT Unit Tests" -Tags "CI" {
             $content.Contains($formatViewDefinition[$i].Name) | Should -BeTrue
         }
     }
-}
-
-Describe "Export-FormatData" -Tags "CI" {
-
-    $testOutput = Join-Path -Path $TestDrive -ChildPath "outputfile"
-
-    AfterEach {
-        Remove-Item $testOutput -Force -ErrorAction SilentlyContinue
-    }
 
-    Context "Check Export-FormatData can be called validly." {
-	    It "Should be able to be called without error" {
-	        { Get-FormatData | Export-FormatData -Path $testOutput } | Should -Not -Throw
-	    }
+    It "Should have a valid xml tag at the start of the file" {
+        $fd | Export-FormatData -Path $testOutput
+        $piped = Get-Content $testOutput -Raw
+        $piped[0] | Should -BeExactly "<"
     }
 
-    Context "Check that the output is in the correct format" {
-	    It "Should not return an empty xml file" {
-	        Get-FormatData | Export-FormatData -Path $testOutput
-	        $piped = Get-Content $testOutput
-	        $piped | Should -Not -BeNullOrEmpty
-	    }
+    It "Should well format output xml" {
```

Comment:
I think the test description should be `Should pretty print xml output` as `pretty print` is a well known term and `well format` may be confused with `well formed` which is another well known xml term.

Reply:
Fixed.